pkgs/development/compilers/dotnet/update.nix

```diff
@@ -129,6 +129,7 @@ writeScript "update-dotnet-vmr.sh" ''
               map(select(.PrivateSourceBuiltArtifactsUrl))
               | .[] | .PrivateSourceBuiltArtifactsUrl' eng/Versions.props)
       fi
+      artifactsUrl="''${artifactsUrl/dotnetcli.azureedge.net/builds.dotnet.microsoft.com}"
```

Comment:
we should maybe leave a comment indicating this is temporary until the release tarballs are fixed?

Reply:
Given they won't use these URLs ever again, I don't know if it is really temporary.

They said they don't know for how long they'll be able to use the `*.azureedge.net` domain for.

Reply:
I just mean presumably they will update the urls in the VMR repo in the future, so we won't need to replace them.

Reply:
True, I was thinking more of previous versions but we don't need to keep supporting those now that I think of it.